addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -234,7 +234,177 @@ class GVAR(actions) {
     };
-
+    
+    // - Airways --------------------------------------------------------------
+    class ChestSeal: BasicBandage {
+        displayName = CSTRING(Actions_ChestSeal);
+        displayNameProgress = CSTRING(Actions_ChestSealProgress);
+        icon = "";
+        category = "airway";
+        treatmentLocations = TREATMENT_LOCATIONS_ALL;
+        allowedSelections[] = {"Body"};
+        allowSelfTreatment = 1;
+        medicRequired = 0;
+        treatmentTime = "5";
+        condition = QFUNC(canChestSeal);
+        callbackSuccess = QFUNC(chestSeal);
+        consumeItem = 1;
+        litter[] = {};
+        items[] = {"ACE_chestSeal"};
+    };
+    
+    class HeadTilt: BasicBandage {
+        displayName = CSTRING(Actions_HeadTilt);
+        displayNameProgress = CSTRING(Actions_HeadTiltProgress);
+        icon = "";
+        category = "airway";
+        treatmentLocations = TREATMENT_LOCATIONS_ALL;
+        allowedSelections[] = {"Head"};
+        allowSelfTreatment = 1;
+        medicRequired = 0;
+        treatmentTime = "5";
+        condition = QFUNC(canHeadTilt);
+        callbackSuccess = QFUNC(airwayAdjunct);
+        callbackProgress = QFUNC(airwayAdjunctProgress);
+        consumeItem = 0;
+        litter[] = {};
+        items[] = {};
+    };
+    
+    class BasicAirway: BasicBandage {        
+        displayName = CSTRING(Actions_BasicAirway);
+        displayNameProgress = CSTRING(Actions_BasicAirwayProgress);
+        icon = "";
+        category = "airway";
+        treatmentLocations = TREATMENT_LOCATIONS_ALL;
+        allowedSelections[] = {"Head"};
+        allowSelfTreatment = 0;
+        medicRequired = 0;
+        treatmentTime = "5";
+        condition = QFUNC(canBasicAirway);
+        callbackSuccess = QFUNC(airwayAdjunct);
+        callbackProgress = QFUNC(airwayAdjunctProgress);
+        consumeItem = 1;
+        litter[] = {};
+        items[] = {"ACE_basicAirway"};
+    };
+    
+    class AdvancedAirway: BasicBandage {
+        displayName = CSTRING(Actions_AdvancedAirway);
+        displayNameProgress = CSTRING(Actions_AdvancedAirwayProgress);
+        icon = "";
+        category = "airway";
+        treatmentLocations = TREATMENT_LOCATIONS_ALL;
+        allowedSelections[] = {"Head"};
+        allowSelfTreatment = 0;
+        medicRequired = 0;
+        treatmentTime = "5";
+        condition = QFUNC(canAdvancedAirway);
+        callbackSuccess = QFUNC(airwayAdjunct);
+        callbackProgress = QFUNC(airwayAdjunctProgress);
+        consumeItem = 1;
+        litter[] = {};
+        items[] = {"ACE_advancedAirway"};
+    };
+    
+    class PulseOximeterOn: BasicBandage {
+        displayName = CSTRING(Actions_PulseOximeterOn);
+        displayNameProgress = CSTRING(Actions_PulseOximeterOnProgress);
+        icon = "";
+        category = "examine";
+        treatmentLocations = TREATMENT_LOCATIONS_ALL;
+        allowedSelections[] = {"Head", "LeftArm", "RightArm", "LeftLeg", "RightLeg"};
```

Comment:
You can't add a pulseoxi on the head, we can discuss limbs, but head is not possible (only earlobe, but that is absolutely Shit)